### ISSUE TITLE
popover for session road map

### ISSUE THREAD

**Issue by rvsp** [zen-class/zen-platform#9]
if date & session details show in a popover, may avoid confusion between regular + additional sessions

**Comment by thecoffeedev** [comment]
@rvsp, are you talking about tooltip for all the sessions in the roadmap?

**Comment by rvsp** [comment]
yes, a tooltip needs to be show for the user when he hovers over the road map numbers
